instance_config.yaml

```diff
@@ -105,7 +105,9 @@ val_username:
 val_transcript_create_url:
 val_video_transcript_status_url:
 
-
+# Celery Worker Config Information
+onsite_worker: False
+celery_threads: 1
```

Comment:
@yro Why we still need these two new variables here? It would be great if we can move these into `static_config.yaml`. I am saying this because the problem is that once the ansible roles will start working this `instance_config.yaml `file be written by ansible, so introduction of any new variable in this file means that we also need to change the ansible role which is very impractical.

Reply:
@yro how are these settings are configurable? i.e. What will be their values on sandbox/production environment? Will their values remain same no matter the environment (sandbox/prod)? if so, we should make them static.

Reply:
Moreover, if these settings are configurable, then let @muhammad-ammar know their environment-specific values.

Reply:
@Qubad786 @muhammad-ammar That's the wrong approach. If this file cannot be configured individually for workers without having to change several repos, this codebase becomes almost unusable. This config file should be copied from the veda-secure repo in whole.

Reply:
@yro veda-secure should only contain secure configs (e.g. secret_keys, access_keys etc), not all the configurations are meant to be secure.

Reply:
@Qubad786 Why make multiple things when there can be one? It seems like having multiple files that need to be changed per instance and use case make the service less operable.

Reply:
So, `static_config.yaml`  should contain non-secure + static configurations. I hope this clears up our perspective.

Reply:
So, let's say I have a larger worker instance I want to deploy. Right now I change a single file. You're saying that I should have to change two files.

Reply:
@yro You can merge the code the way best suitable as we want to release the work as soon as possible. 

But we need to resolve this. I have explained the reason in my first comment above. Can you please spare some time and explain why we need to change/copy this file or any other file every time. It would be great if you can share the details, then I can discuss here with my team and after we can have a discussion with you to improve this whole flow. 
What do you say?

Reply:
@yro the answer to your question lies in @muhammad-ammar 's first comment on the top of the conversation.

Reply:
@muhammad-ammar I'll move these to static, so as to unblock you -- they can be changed later if we want.

Reply:
@yro I am not blocked on anything. I divided the config into two for ansible work. But that work will take time to finish. So you can do thing they way easy for you now. As I said in my last comment we will discuss about this.

Reply:
👍 Sounds good! We will discuss.